board.rb

```diff
@@ -1,89 +1,74 @@
 class Board
-  attr_reader :person, :level
-
-  PERSON  = "@"
-  CRATE   = "o"
-  WALL    = "#"
-  OPEN    = " "
-  STORAGE = "."
-  PERSON_ON_STORAGE = "+"
-  CRATE_ON_STORAGE  = "*"
+  attr_reader :person, :level, :cells
 
   def initialize(level:)
     @level = level
     @person = Person.new(self)
-    make_board
+    create_board
   end
 
-  def make_board
+  def create_board
     lines = File.readlines("sokoban_levels.txt")
     levels = lines.map { |line| line.chomp.ljust(19) }.join("\n")
     levels = levels.split(/\n {19}\n/)
 
-    @cells = levels[level - 1]
+    @cells = levels[level - 1].chars.map { |char| Cell.new(self, char) }
   end
 
   def to_s
-    cells
+    board_cells = cells.map {|c| c.occupant}
```

Comment:
Space between { and | missing.<br>Space missing inside }.<br>Pass `&:occupant` as an argument to `map` instead of a block.